Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/zip.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/zsh
+SCRIPT=$(realpath "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd "$SCRIPTPATH"
+
+TEMP_DIR="OUTPUT_DIR"
+
```

Comment:
Do you want to clean the temp directory first?

Reply:
done